cli/lib/kontena/cli/certificate/list_command.rb

```diff
@@ -15,7 +15,7 @@ class ListCommand < Kontena::Command
     THREE_DAYS = 3 * 24 * 60 * 60
 
     def fields
-      quiet? ? ['subject'] : {subject: 'subject', "expiration" => 'expires_in'}
+      quiet? ? ['subject'] : {subject: 'subject', "expiration" => 'expires_in', auto_renewable?: 'auto_renewable'}
```

Comment:
The exact formatting here might not be the best, but it works. We can fix it later if it's too ugly:

```
SUBJECT                        EXPIRATION   AUTO_RENEWABLE?
⊛ test.188.226.131.23.xip.io   77 days      true
⊛ test.188.226.131.23.nip.io   67 days      false
```

Reply:
`AUTO_RENEWABLE?` this looks a bit ugly but we can fix it later.